composer.json

```diff
@@ -22,7 +22,7 @@
         "nikic/php-parser": "^4.13.2",
         "phar-io/version": "^3.2",
         "phpstan/phpdoc-parser": "^1.2",
-        "phpstan/phpstan": "^1.5.6",
+        "phpstan/phpstan": "1.5.*",
```

Comment:
this will make existing user with phpstan 1.6 already cannot update. 

The phpstan notice need to be resolved or ignored instead.

Reply:
I see, thanks for the info. I will take another look when I have time. :+1: